packages/components/src/card/card-footer/component.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { contextConnect } from '../../ui/context';
+import { Flex } from '../../flex';
+import { useCardFooter } from './hook';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').FooterProps, 'div'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function CardFooter( props, forwardedRef ) {
```

Comment:
Wonder if this (and other subcomponents) could use the `createComponent` function instead?

Reply:
Not necessary, we can leave it as is